README.md

```diff
@@ -98,7 +98,7 @@ Permissions that applications may use (names are subject to change):
 | Ambience   | Set and edit ambiences. |
 | AppLaunch  | Launching and stopping systemd services. This is usually needed for background tasks. |
 | ApplicationInstallation | Installing and uninstalling applications. |
-| Audio | Playing audio, changing audio configuration and showing audio controls on lockscreen. |
+| Audio | Playing audio, changing audio configuration, recording via microphone and showing audio controls on lockscreen. |
```

Comment:
I would perhaps write this here in the way we intend it to work. I.e. no recording at all.

Reply:
Well hiding the recording part is kind of misleading in a way, as that is the reality until the streams can be separated one way or another. Maybe just adding that recording in parenthesis with a note "until Pulseaudio streams can be separated" as that is intended to be in the actual permission file. I see no reason to document it here as well as this is what the developer should really read.

Reply:
Removed that in 36f9ad3 but I'm not really happy with it. It may become as a surprise to devs when only using `Audio` permission also mic starts to work.